qa/rpc-tests/feature_issuance.py

```diff
@@ -316,6 +316,82 @@ def run_test(self):
+        blind_tx = self.nodes[0].blindrawtransaction(reissued_tx["hex"])
+        signed_tx = self.nodes[0].signrawtransaction(blind_tx)
+        tx_id = self.nodes[0].sendrawtransaction(signed_tx["hex"])
+        self.nodes[0].generate(1)
+        assert_equal(self.nodes[0].gettransaction(tx_id)["confirmations"], 1)
+
+        # Now send reissuance token to blinded multisig, then reissue
+        addrs = []
+        for i in range(3):
+            addrs.append(self.nodes[0].validateaddress(self.nodes[0].getnewaddress())["pubkey"])
+
+
+        multisig_addr = self.nodes[0].createmultisig(2,addrs)
+        blinded_addr = self.nodes[0].getnewaddress()
+        blinding_pubkey = self.nodes[0].validateaddress(blinded_addr)["confidential_key"]
+        blinding_privkey = self.nodes[0].dumpblindingkey(blinded_addr)
+        blinded_multisig = self.nodes[0].createblindedaddress(multisig_addr["address"], blinding_pubkey)
+        # Import address so we consider the reissuance tokens ours
+        self.nodes[0].importaddress(blinded_multisig)
+        # Import blinding key to be able to decrypt values sent to it
+        self.nodes[0].importblindingkey(blinded_multisig, blinding_privkey)
+
+        self.nodes[0].sendtoaddress(blinded_multisig, self.nodes[0].getbalance()[issued_asset["asset"]], "", "", False, issued_asset["asset"])
+        self.nodes[0].generate(1)
+
+        # Get that multisig output
+        utxo_info = None
+        # Find info about the token output using wallet
+        for utxo in self.nodes[0].listunspent():
+            if utxo["asset"] == issued_asset["token"]:
+                utxo_info = utxo
+                break
+        assert(utxo_info is not None)
+
+        # Now make transaction spending that input
+        raw_tx = self.nodes[0].createrawtransaction([], {issued_address:1}, 0, {issued_address:issued_asset["token"]})
+        funded_tx = self.nodes[0].fundrawtransaction(raw_tx)["hex"]
+        # Find the reissuance input
+        reissuance_index = -1
+        for i, tx_input in enumerate(self.nodes[0].decoderawtransaction(funded_tx)["vin"]):
+            if tx_input["txid"] == utxo_info["txid"] and tx_input["vout"] == utxo_info["vout"]:
+                reissuance_index = i
+                break
+        assert(reissuance_index != -1)
+        reissued_tx = self.nodes[0].rawreissueasset(funded_tx, [{"asset_amount":3, "asset_address":self.nodes[0].getnewaddress(), "input_index":reissuance_index, "asset_blinder":utxo_info["assetblinder"], "entropy":issued_asset["entropy"]}])
+
+        blind_tx = self.nodes[0].blindrawtransaction(reissued_tx["hex"])
+        signed_tx = self.nodes[0].signrawtransaction(blind_tx)
+        tx_id = self.nodes[0].sendrawtransaction(signed_tx["hex"])
+        self.nodes[0].generate(1)
+        assert_equal(self.nodes[0].gettransaction(tx_id)["confirmations"], 1)
+
+        # TODO test raw issuance for non-empty contract hash
```

Comment:
I think you did that.. Or do you mean raw *RE*issuance with non-empty contract hash?